org.lflang/build.gradle

```diff
@@ -106,6 +107,7 @@ task updateRustRuntime {
     description('Record the VCS revisions of the language runtimes into a properties file available at runtime.')
 
     dependsOn getSubmoduleVersions
+    inputs.files(getSubmoduleVersions.outputs)
```

Comment:
But if the output of the `git submodule status` did not change (the getSubmoduleVersions task), then this task is considered up to date.

Reply:
```suggestion
    // If the output of the git submodule status did not change (the getSubmoduleVersions task), then this task is considered up to date.
    inputs.files(getSubmoduleVersions.outputs)
```